### ISSUE TITLE
Predictions of the Misclassified instances

### ISSUE THREAD

**Issue by forgit** [biolab/orange3#3405]
##### Orange version

3.18.0 from conda-forge

##### Expected behavior
Predicted classes (Predictions widget) for the **misclassified** instances (from the Confusion Matrix) have to be all wrong.

##### Actual behavior
In fact some of the predicted classes for the misclassified instances taken from the confusion matrix are right. However, Data Table widget shows there are two columns in the output from Predictions widget. The first column has a name "attribute(Logistic Regression)" and it shows all instances as misclassified (which is a proper behaviour). The second column "Logistic Regression" is the one that matches the Predictions widget view.

##### Steps to reproduce the behavior
See screenshot attached. It is a standard dataset.

![screen shot 2018-11-16 at 4 31 23 pm](https://user-images.githubusercontent.com/3049104/48649295-28a9a800-e9c0-11e8-91fe-2e0bcf6ceffa.png)

**Comment by janezd** [comment]
Does Test & Score use Cross validation? In this case, this is not the same model. In Confusion matrix you see (mis)classifications of test data by 10 different models, built from 10 different subsets of training data (assuming you run 10-fold cross validation). The model that goes into Predictions is built from the entire data set. It is also tested on this same data, which explains its accuracy (which comes from overfitting).

Please check and confirm whether this is the case.

**Comment by forgit** [comment]
Thank you @janezd . Yes, I used 10-fold CV. 

I confirm that the results are accurate when testing on a test set from the Data Sampler widget.
